ert_gui/simulation/view/iteration.py

```diff
@@ -20,13 +20,25 @@ def __init__(self, iter: int, parent=None) -> None:
         self._real_view.clicked.connect(self._select_real)
 
         self._job_view = QTableView(self)
+        self._job_view.clicked.connect(self.job_clicked)
 
         layout = QVBoxLayout()
         layout.addWidget(self._real_view)
         layout.addWidget(self._job_view)
 
         self.setLayout(layout)
 
+    @Slot(QModelIndex)
+    def job_clicked(self, index):
+        node = index.internalPointer()
+        if node is None or node.type != NodeType.JOB:
+            return
+
+        if index.column() == 6:
+            print(node.data.get("stdout"))
+        elif index.column() == 7:
+            print(node.data.get("stderr"))
+
```

Comment:
Should of course instantiate a `FileDialog` as in https://github.com/equinor/ert/blob/a7a5263dca9c00e30c1434a320998b7bdb2c1e24/ert_gui/simulation/detailed_progress.py#L230